corehq/apps/translations/integrations/transifex/transifex.py

```diff
@@ -71,16 +71,14 @@ def send_translation_files(self):
 
     @cached_property
     def client(self):
-        transifex_account_details = settings.TRANSIFEX_DETAILS
-        if transifex_account_details:
-            return TransifexApiClient(
-                transifex_account_details['token'],
-                transifex_account_details['organization'],
-                self.project_slug,
-                self.use_version_postfix,
-            )
-        else:
-            raise Exception(_("Transifex account details not available on this environment."))
+        transifex_project = TransifexProject.objects.filter(slug=self.project_slug).first()
```

Comment:
since `project_slug` isn't necessarily unique shouldn't this also filter by domain?

Reply:
making it unique

Reply:
surely different ors on transifex could have the same project name?

Reply:
I'd also use `.get` rather than `.filter().first` since the expectation here is that it does exist and is unique.

Reply:
@snopoke do you mean different slug but with different project name?
Thats possible to create on transifex. In view when selecting project we are showing both name and the slug so one can be sure of what they are selecting.